### PR TITLE
Add strategies doc

### PR DESCRIPTION
**Pull request by jbendwick**

### CODE REVIEW

**Comment by cdasch** [comment]
Ok this looks good!

**Comment by lward1** [comment]
how do I reveiw

File: STRATEGIES.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+[Story](https://www.pivotaltracker.com/story/show/153708395)
+------------------------------------------------------------
+
+# How do I make a pull request?
+step 1: go to github
+Step 2: click the "pull requests" tab in top left corner (https://github.com/altereagle/trinity-carpool/pulls)
+Step 3: click the green "new pull request" button
+step 4:change the "base" and "compare" sections to what is needed
+Step 5:
+step 6:
```
Comment (ironhorse2023) [inline STRATEGIES.md]: need to either put in these steps or take these out

Comment (cdasch) [reply STRATEGIES.md]: Ya we should take them out

Comment (ironhorse2023) [reply STRATEGIES.md]: step 5 and 6

File: STRATEGIES.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+Step _: DONT MERGE THE PULL REQUEST BEFORE ALL REVIEWS ARE COMPLETED!!!!!! Plus, don't have more than one person do it becasue then you're comutor will break
```
Comment (jbendwick) [inline STRATEGIES.md]: step is blank

File: STRATEGIES.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
```
Comment (ironhorse2023) [inline STRATEGIES.md]: we should take this out too

**Pull request merged**